hadoop-hdds/docs/themes/ozonedoc/static/swagger-resources/recon-api.yaml

```diff
@@ -229,6 +233,229 @@ paths:
               schema:
                 $ref: '#/components/responses/DeletedMismatchedContainers'
+
+  /volumes:
+    get:
+      tags:
+        - Volumes
+      summary: Fetches all volumes in the cluster.
+      operationId: getVolumes
+      parameters:
+        - name: prevKey
+          in: query
+          description: Only returns the volume after the given prevKey.
+          required: false
+          schema:
+            type: string
+            example: vol1
+        - name: limit
+          in: query
+          description: Only returns the limited number of results. The default limit is 1000.
+          required: false
+          schema:
+            type: integer
+            default: 1000
+      responses:
+        '200':
+          description: Successful operation
+          content:
+            application/json:
+              schema:
+                type: object
+                properties:
+                  totalCount:
+                    type: integer
+                    example: 4
+                  volumes:
+                    type: array
+                    items:
+                      type: object
+                      properties:
+                        volume:
+                          type: string
+                          example: vol1
+                        owner:
+                          type: string
+                          example: testuser
+                        admin:
+                          type: string
+                          example: ozone
+                        creationTime:
+                          type: integer
+                          example: 1665588176660
+                        modificationTime:
+                          type: integer
+                          example: 1665590397315
+                        quotaInNamespace:
+                          type: integer
+                          example: 2048
+                        quotaInBytes:
+                          type: integer
+                          example: 1073741824
+                        usedNamespace:
+                          type: integer
+                          example: 10
+                        acls:
+                          type: array
+                          items:
+                            type: object
+                            properties:
+                              type:
+                                type: string
+                                example: USER
+                              name:
+                                type: string
+                                example: testuser
+                              scope:
+                                type: string
+                                example: ACCESS
+                              aclList:
+                                type: array
+                                items:
+                                  type: string
+                                  example: WRITE
+              example:
```

Comment:
We do not require an example section as it will pick up the examples from the schema section